src/foam/nanos/fs/TextSaveView.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.fs',
+  name: 'TextSaveView',
+  extends: 'foam.u2.Element',
+
+  documentation: 'view to save plain text as file',
+
+  requires: [
+    'foam.blob.BlobBlob',
+    'foam.nanos.fs.File'
+  ],
+
+  properties: [
+    'data',
+    {
+      name: 'fileName'
+    }
+  ],
+
+  methods: [
+    function initE() {
+      this.SUPER();
+      var name = "";
+
+      this
+        .start('p')
+          .add('File Name')
+        .end()
+        .start('input')
```

Comment:
You could just do .add(this.FILE_NAME) and achieve the same result with having to add the onBlur listener. Just make this class extend foam.u2.Controller instead.